sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/directconnectivity/ConsistencyWriterTest.java

```diff
@@ -303,7 +303,10 @@ public void storeResponseRecordedOnException(Exception ex, StoreResponse storeRe
         TimeoutHelper timeoutHelper = Mockito.mock(TimeoutHelper.class);
         RxDocumentServiceRequest dsr = mockDocumentServiceRequest(clientContext);
 
-        consistencyWriter.writeAsync(dsr, timeoutHelper, false).subscribe();
+        consistencyWriter.writeAsync(dsr, timeoutHelper, false).onErrorResume(throwable -> {
+            //  do nothing
+            return Mono.empty();
```

Comment:
if we fail over, should we fail the test ?

Reply:
We were not failing earlier, because it was `subscribe()` - test could have finished before we fail or succeed this method, however, we can. I will put it in next iteration.

Reply:
fixed.